plasTeX/Imagers/__init__.py

```diff
@@ -636,12 +638,17 @@ def compileLatex(self, source):
                         cmd_line))
         output = None
         for ext in ['.dvi','.pdf','.ps']:
-            if os.path.isfile('images'+ext):
-                output = open('images'+ext, 'rb')
+            try:
+                with open('images'+ext, 'rb') as f:
+                    output = open('images'+ext, 'rb').read()
                 break
+            except OSError:
+                # Ignore if file not found
+                pass
 
         # Change back to original working directory
         os.chdir(cwd)
+        shutil.rmtree(tempdir, True)
```

Comment:
This is yet another unrelated change, right? It is often useful to keep those temporary stuff for debugging, and I don't think they are so large (and people are still free to wipe out `/tmp` from times to times).

Reply:
I also use this for debugging sometimes, but it is less helpful if the `/tmp/` is full of things called `tmp....`. Personally, I use this on a shared host, and it would be irresponsible to pollute `/tmp/` with all my stuff. I'll add an option that removes the temp directories by default.

Reply:
Ok, good.